packages/pyright-internal/src/analyzer/cacheManager.ts

```diff
@@ -8,6 +8,9 @@
  * if memory usage approaches the max heap space.
  */
 
+import { HeapInfo } from 'v8';
```

Comment:
Will this break browser support for vscode.dev?

Reply:
I don't know yet.  I was going to handle this when we port this to Pylance. If that's the case, I can just import the type. Actually I'll just do that now.